src/GafferSceneUI/Inspector.cpp

```diff
@@ -205,7 +252,7 @@ void Inspector::inspectHistoryWalk( const GafferScene::SceneAlgo::History *histo
 				{
 					// We've found the source of the value we're inspecting.
 
-					result->m_source = static_cast<ValuePlug *>( spreadsheetAwareSource( source.get() ) );
+					result->m_source = static_cast<ValuePlug *>( userEditableSource( source.get() ) );
```

Comment:
Now we're making spreadsheet-defaults non-editable by other means, we can make the inspector more useful by _not_ storing null here. Then `Result::source()` will return the source that we've discovered (information is good), but `Result::editable()` will still stop us from editing it. It also simplifies the rest of this block of code if we don't need to consider the null case.